login_test.py

```diff
@@ -116,26 +121,27 @@ def test_cantLoginWithBadCredentials():
     #wait until the error message appears, or 10 seconds, whichever is shorter
     WebDriverWait(driver, 10).until(EC.presence_of_element_located((By.XPATH, "//p[@class='error']")))
 
-    assert "forkert brugernavn eller password" in driver.page_source
-
+    source_to_check = driver.page_source
     driver.quit()
 
+    assert "forkert brugernavn eller password" in source_to_check
+
+
+
 def test_canLoginWithGoodCredentials(dummy_user):
-    global dummy_user_nickname
-    global dummy_user_password
+    global dummy_user_email, dummy_user_password
     driver = webdriver.Firefox()
     driver.get(pages["login"])
 
     username_entry = driver.find_element_by_id("input_username")
-    username_entry.send_keys(dummy_user_nickname)
+    username_entry.send_keys(dummy_user_email)
 
     password_entry = driver.find_element_by_id("input_password")
     password_entry.send_keys(dummy_user_password)
     password_entry.submit()
 
-    import time
-    time.sleep(3)
     #wait until redirected to new page, or 10 seconds, whichever is shorter

```

Comment:
I don't think this comment is accurate any more.

Reply:
(or at least, is explaining the line that is obvious but not the line that isn't...)